torchao/dtypes/affine_quantized_tensor_ops.py

```diff
@@ -490,10 +492,50 @@ def _(func, types, args, kwargs):
         self.quant_max,
         self.zero_point_domain,
         dtype=self.dtype,
-        strides=self.stride(),
+        strides=self.stride() if len(block_size)==2 else None,
```

Comment:
why this?

Reply:
the stride can change for 3 dimensional tensors when doing slicing and indexing ops.